expect/_types.ts

```diff
@@ -915,3 +915,5 @@ export interface OldSnapshotPlugin {
   ) => string;
   test: Test;
 }
+
+export type SnapshotPlugin = NewSnapshotPlugin | OldSnapshotPlugin;
```

Comment:
Is `OldSnapshotPlugin` still common? How about only supporting `NewSnapshotPlugin`?

Reply:
Yes, I checked some serialize plugins, they still use `OldSnapshottPlugin` type definition, like:

[jest-serializer-vue](https://github.com/eddyerburgh/jest-serializer-vue/blob/master/index.js#L17)
[jest-serializer-html](https://github.com/algolia/jest-serializer-html/blob/master/createSerializer.js#L17)
[jest-serializer-react-helmet](https://github.com/keplersj/jest-serializer-react-helmet/blob/master/src/index.tsx#L24)
[jest-snapshot-serializer-ansi](https://github.com/ikatyang/jest-snapshot-serializer-ansi/blob/master/src/index.ts#L8)

Reply:
Fair enough